query/parse.go

```diff
@@ -124,7 +125,11 @@ func parseExpr(in []byte) (Q, int, error) {
 		}
 		expr = &caseQ{text}
 	case tokRepo:
-		expr = &Repo{Pattern: text}
+		if text == "" {
+			return nil, 0, fmt.Errorf("the repo: expr must have an argument")
+		}
+
+		expr = &Repo{regexp.MustCompile(text)}
```

Comment:
you can handle the error here and return it rather than using MustCompile which can panic.